upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -211,6 +211,61 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
+state_name="REMOVE_DUPLICATE_BMC_DNS_RECORDS"
+state_recorded=$(is_state_recorded "${state_name}" $(hostname))
+if [[ $state_recorded == "0" ]]; then
+    echo "====> ${state_name} ..."
+    {
+        # Find a pod that works.
+        # List names of all Running vault pods, grep for just the cray-vault-# pods, and try them in
+        # turn until one of them has the IPMI credentials.
+        USERNAME=""
+        IPMI_PASSWORD=""
+        for VAULT_POD in $(kubectl get pods -n vault --field-selector status.phase=Running --no-headers \
+                            -o custom-columns=:.metadata.name | grep -E "^cray-vault-(0|[1-9][0-9]*)$") ; do
+        USERNAME=$(kubectl exec -it -n vault -c vault ${VAULT_POD} -- sh -c \
```

Comment:
This isn't working for me when I test it on hela.   Where does VAULT_TOKEN from from?

Reply:
Lines 226-236 within the for loop should be indented.